src/commands/createNewProject/initProjectForVSCode.ts

```diff
@@ -89,12 +84,12 @@ async function writeDebugConfiguration(projectCreator: ProjectCreatorBase, vscod
     }
 }
 
-async function writeVSCodeSettings(projectCreator: ProjectCreatorBase, vscodePath: string, language: string, templateFilter: string): Promise<void> {
+async function writeVSCodeSettings(projectCreator: ProjectCreatorBase, vscodePath: string, language: string, templateFilter: string, runtime: ProjectRuntime): Promise<void> {
```

Comment:
nit: Can't you call `getRuntime()` from projectCreator rather than having to pass it in?

Reply:
I tried to use getRuntime as little as possible just because:
1. It's async
1. It will prompt if runtime hasn't been defined yet, and I want to make it somewhat clear when that prompt might happen